database/db_manager/db_manager.py

```diff
@@ -258,12 +258,12 @@ def handler(event: Event, _) -> Response:
     response: Response = {"statusCode": 200, "body": json.dumps("")}
     db_helper = DatabaseHelper()
 
-    event_type = event.get("event_type", EventType.CREATE_DB.value)
-    if event_type == EventType.CREATE_DB.value:
+    event_type = event.get("action", Action.CREATE_DB.value)
```

Comment:
I would try to convert the action string to an actual Action enum object somehow. Maybe something like this:
```suggestion
    try:
        event_type = Action(event["action"])
    except ValueError, KeyError:
        event_type = Action.CREATE_DB
```
Then there won't be no need to use `.value` in comparisons.

Reply:
Good point, I was wondering what is the convention for comparing enums with incoming data.